### DISCUSSION TITLE
Override default styles that come with model viewer

### DISCUSSION THREAD

**Discussion by devhims** [google/model-viewer#2040] [Q&A]
Is there a way for us to override all the default styles that model viewer brings with it so that we can do it from scratch?

**Comment by elalish** [comment]
Yes, you override our defaults using [slots](https://modelviewer.dev/docs/#loading-slots). If you use our [editor](https://modelviewer.dev/editor/) and export the ZIP, it will contain some custom slotted UX to give you a starting point for making your own.
